app/src/main/res/layout/feedback_item.xml

```diff
@@ -17,7 +17,7 @@
   <FrameLayout
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:background="@{viewModel.hasConversationView ? @drawable/content_blue_background : @drawable/transparent_background}"
+    android:background="@{viewModel.isAudioPlaying ? @drawable/content_yellow_background : @drawable/content_blue_background}"
```

Comment:
Changes in xml files does affect the question player too. It does mess up the entire question player/ practice mode.